Library/Homebrew/formulary.rb

```diff
@@ -237,18 +245,73 @@ def self.loader_for(ref)
     return NullLoader.new(ref)
   end
 
+  def self.find_with_priority(ref, is_installing, warn_all_ambiguity)
```

Comment:
This should instead take an `options` hash and use the last two parameter names as hash keys. This will improve readability of callers.